src/CarouselIndicators.js

```diff
@@ -13,6 +13,7 @@ const CarouselIndicators = (props) => {
     ), cssModule);
     return (
       <li
+        {...item.props}
```

Comment:
The main purpose of this change is to introduce ids into the carousel indicators?

Reply:
@kyletsang ids and data-testid to every indicator, to get indicator through testid and fire events on them.

Reply:
@kyletsang when this PR will gets merged and new release will be published?

Reply:
I guess my concern here would be the naming of `props`.  The usage of carousel is pretty open-ended, so could potentially pass an array of components as `items` to `CarouselIndicators` which will conflict.  Maybe it's better to allow users to pass children into `CarouselIndicators` so they can control the rendering?

Thoughts @TheSharpieOne?